Python/flowgraph.c

```diff
@@ -481,9 +481,12 @@ no_redundant_jumps(cfg_builder *g) {
             if (IS_UNCONDITIONAL_JUMP_OPCODE(last->i_opcode)) {
                 basicblock *next = next_nonempty_block(b->b_next);
                 basicblock *jump_target = next_nonempty_block(last->i_target);
-                assert(jump_target != next);
                 if (jump_target == next) {
-                    return false;
+                    assert(next);
+                    if (last->i_loc.lineno == next->b_instr[0].i_loc.lineno) {
+                        assert(0);
```

Comment:
Is this `assert(0)` a leftover debugging help? Shouldn't it be removed before the final merge?

Reply:
Sorry for the noise. I've just realized the old `assert(jump_target != next);` so presumably this should stay?

Reply:
Yes, this function is only for debugging and it's nice if it crashes where the problem is.